Description: Don't build-depend on libgcrypt, as nothing is used from it
Author: Didier Raboud <odyx@debian.org>
Bug-Debian: https://bugs.debian.org/638416
Bug: http://www.cups.org/str.php?L????

[CUPS Ticket #4399] -- http://www.cups.org/str.php?L4399

Backported from http://www.cups.org/strfiles.php/3308/cups-no-gcrypt.patch

Upstream-Status: Backport 

Signed-off-by: Armin Kuster <akuster@mvista.com>

Last-Update: 2014-04-07
--- a/config-scripts/cups-ssl.m4
+++ b/config-scripts/cups-ssl.m4
@@ -66,7 +66,6 @@
     dnl Then look for GNU TLS...
     if test $have_ssl = 0 -a "x$enable_gnutls" != "xno" -a "x$PKGCONFIG" != x; then
     	AC_PATH_PROG(LIBGNUTLSCONFIG,libgnutls-config)
-    	AC_PATH_PROG(LIBGCRYPTCONFIG,libgcrypt-config)
 	if $PKGCONFIG --exists gnutls; then
 	    have_ssl=1
 	    SSLLIBS=`$PKGCONFIG --libs gnutls`
@@ -84,14 +83,6 @@
 	if test $have_ssl = 1; then
 	    CUPS_SERVERCERT="ssl/server.crt"
 	    CUPS_SERVERKEY="ssl/server.key"
-
-            if $PKGCONFIG --exists gcrypt; then
-	        SSLLIBS="$SSLLIBS `$PKGCONFIG --libs gcrypt`"
-	        SSLFLAGS="$SSLFLAGS `$PKGCONFIG --cflags gcrypt`"
-	    elif test "x$LIBGCRYPTCONFIG" != x; then
-	        SSLLIBS="$SSLLIBS `$LIBGCRYPTCONFIG --libs`"
-	        SSLFLAGS="$SSLFLAGS `$LIBGCRYPTCONFIG --cflags`"
-	    fi
 	fi
     fi
 
--- a/cups/http-private.h
+++ b/cups/http-private.h
@@ -80,7 +80,6 @@
 #  elif defined HAVE_GNUTLS
 #    include <gnutls/gnutls.h>
 #    include <gnutls/x509.h>
-#    include <gcrypt.h>
 #  elif defined(HAVE_CDSASSL)
 #    include <CoreFoundation/CoreFoundation.h>
 #    include <Security/Security.h>
